weheat/abstractions/heat_pump.py

```diff
@@ -118,25 +157,30 @@ def cop(self):
 
     @property
     def inside_unit_water_pump_state(self):
+        """Decoded water pump state."""
         return self._if_available("control_bridge_status_decoded_water_pump")
 
     @property
     def inside_unit_auxilary_pump_state(self):
+        """Decoded auxilary pump state."""
         return self._if_available("control_bridge_status_decoded_water_pump2")
 
     @property
     def inside_unit_dhw_valve_or_pump_state(self):
+        """Decoded dhw valve or pump state."""
         return self._if_available("control_bridge_status_decoded_dhw_valve")
```

Comment:
```suggestion
    def inside_unit_dhw_valve_or_pump_state(self):
        """Decoded DHW valve or pump state."""
        return self._if_available("control_bridge_status_decoded_dhw_valve")
```

DHW capitalized for consistency?